packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.1
+    tag: v1.0.3-rc1
```

Comment:
This will be changed once the final image is out, might even be removed from this PR

Reply:
removing the image changes, will be done through @briandowns 's PR